typing/btype.ml

```diff
@@ -281,7 +282,7 @@ let fold_row f init row =
       (row_fields row)
   in
   match get_desc (row_more row) with
-  | Tvar _ | Tunivar _ | Tsubst _ | Tconstr _ | Tnil
+  | Tvar _ | Tunivar _ | Tsubst _ | Tconstr _ | Tquote _ | Tsplice _ | Tnil
```

Comment:
Can you try to move this to the `asssert false` branch instead? I think `Tquote` and `Tsplice` shouldn't appear as row variable.

Reply:
I think so too.